custom_components/hubitat/event.py

```diff
@@ -0,0 +1,100 @@
+"""Hubitat event entities."""
+
+from logging import getLogger
+from typing import Unpack
+
+from homeassistant.components.event import (
+    EventDeviceClass,
+    EventEntity,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import (
```

Comment:
Only `_TRIGGER_ATTR_MAP` is actually needed in here; everything that's checking if an attribute is in `_TRIGGER_ATTRS` could instead be checking if it's in `_TRIGGER_ATTR_MAP`.

However, instead of importing these constants, it would be a bit cleaner to define a constant in this module that applies specifically to button event entities, like
```py
ATTR_EVENTS = {
  DeviceAttribute.DOUBLE_TAPPED: EventName:DOUBLE_TAPPED,
  DeviceAttribute.PUSHED: EventName.PUSHED,
  ...
}
```

This should include at least the 4 attributes and events used by Hubitat (double tapped, held, pushed, and released).

Then `handle_event` can do `if event.attribute not in ATTR_EVENTS` and `event_type = ATTR_EVENTS[event.attribute]`. It just makes the code in here a little easier to follow.